processes/UofM.py

```diff
@@ -67,7 +67,7 @@ def addHasPartMapping(self, resultsRecord, record):
                 urlPDFObject,
                 'UofM',
                 'application/pdf',
-                '{"catalog": false, "download": true, "reader": false, "embed": false, "nypl_login": true}'
+                '{"catalog": false, "download": true, "reader": false, "embed": false}'
```

Comment:
could the nypl_login boolean be passed in from the CSV or JSON file?

Reply:
No it cannot since these flags are created manually by us in the backend so Jackie on the frontend knows how these books are formatted on the site based on these flags. the ingestJSONfiles folder is there for times that I have manually ingested books from sources such as University of Michigan before the automatic ingest process has been created.